examples/sas_logical_interconnect.rb

```diff
@@ -80,9 +76,9 @@
   client my_client
   firmware 'ROM Flash - SPP'
   firmware_data(
-    ethernetActivationDelay: 15,
-    fcActivationDelay: 20,
-    force: true
+    fwActivationMode: 'Parallel',
```

Comment:
I changed the attributes because these attributes do not work with sas logical connection.